### ISSUE TITLE
Pokemon to display at start is blank

### ISSUE THREAD

**Issue by Yombre** [NoxxDev/NecroBot#3]
### Expected Behavior
Pokemon to display at start should show highest CP and highest IV (Perfect) pokemon.

### Actual Behavior
Pokemon to display at start is empty.  (Blanked out IP, Lat & Lng for protection)
```
[10:19:22] (INFO) Your IP is: xxx.xxx.xxx.xxx
[10:19:23] (UPDATE) Perfect! You already have the newest Version 0.8.8
[10:19:23] (INFO) Logging in using Ptc
[10:19:24] (INFO) Playing as xxxxxxxxxxxxxxxx
[10:19:26] (ATTENTION) Detected realistic Traveling , using Default Settings inside config.json
[10:19:26] (ATTENTION) Make sure Lat & Lng are right. Exit Program if not! Lat: xx.xxxxxxxxx Lng: xx.xxxxxxxxx
[10:19:27] (INFO) ====== DisplayHighestsCP ======
[10:19:27] (INFO) ====== DisplayHighestsPerfect ======
[10:19:27] (EVOLVED) Potential Evolutions: 27
```
### Your config.json
```
{
  "TranslationLanguageCode": "en",
  "CheckForUpdates": true,
  "AutoUpdate": false,
  "TransferConfigAndAuthOnUpdate": true,
  "UseWebsocket": false,
  "WebSocketPort": 14251,
  "UseTelegramAPI": false,
  "TelegramAPIKey": null,
  "TelegramPassword": "12345",
  "StartupWelcomeDelay": false,
  "AmountOfPokemonToDisplayOnStart": 10,
  "DetailedCountsBeforeRecycling": true,
  "CatchPokemon": true,
  "MaxBerriesToUsePerPokemon": 3,
  "AutomaticallyLevelUpPokemon": true,
  "OnlyUpgradeFavorites": false,
  "UseLevelUpList": true,
  "AmountOfTimesToUpgradeLoop": 5,
  "GetMinStarDustForLevelUp": 5000,
  "LevelUpByCPorIv": "iv",
  "UpgradePokemonCpMinimum": 1000.0,
  "UpgradePokemonIvMinimum": 99.0,
  "UpgradePokemonMinimumStatsOperator": "and",
  "DisableHumanWalking": false,
  "DefaultLatitude": xx.xxxxxxx,
  "DefaultLongitude": xx.xxxxxxx,
  "WalkingSpeedInKilometerPerHour": 17.38,
  "RandomlyPauseAtStops": true,
  "MaxSpawnLocationOffset": 10,
  "FastSoftBanBypass": false,
  "DelayBetweenPlayerActions": 2287,
  "DelayBetweenPokemonCatch": 8722,
  "DumpPokemonStats": false,
  "EvolveAboveIvValue": 95.0,
  "EvolveAllPokemonAboveIv": true,
  "EvolveAllPokemonWithEnoughCandy": true,
  "EvolveKeptPokemonsAtStorageUsagePercentage": 90.0,
  "KeepPokemonsThatCanEvolve": true,
  "KeepMinCp": 1250,
  "KeepMinIvPercentage": 90.0,
  "KeepMinLvl": 6,
  "KeepMinOperator": "or",
  "UseKeepMinLvl": false,
  "PrioritizeIvOverCp": false,
  "KeepMinDuplicatePokemon": 6,
  "UseGpxPathing": false,
  "GpxFile": "GPXPath.GPX",
  "VerboseRecycling": true,
  "RecycleInventoryAtUsagePercentage": 90.0,
  "RandomizeRecycle": false,
  "RandomRecycleValue": 5,
  "DelayBetweenRecycleActions": false,
  "UseEggIncubators": true,
  "UseEggIncubatorMinKm": 2,
  "UseLuckyEggConstantly": false,
  "UseLuckyEggsMinPokemonAmount": 30,
  "UseLuckyEggsWhileEvolving": true,
  "UseIncenseConstantly": true,
  "UseBerriesMinCp": 1000,
  "UseBerriesMinIv": 90.0,
  "UseBerriesBelowCatchProbability": 0.2,
  "UseBerriesOperator": "or",
  "UseSnipeLocationServer": false,
  "SnipeLocationServer": "localhost",
  "SnipeLocationServerPort": 16969,
  "GetSniperInfoFromPokezz": false,
  "GetOnlyVerifiedSniperInfoFromPokezz": true,
  "GetSniperInfoFromPokeSnipers": true,
  "GetSniperInfoFromPokeWatchers": true,
  "GetSniperInfoFromSkiplagged": true,
  "MinPokeballsToSnipe": 20,
  "MinPokeballsWhileSnipe": 0,
  "MinDelayBetweenSnipes": 60000,
  "SnipingScanOffset": 0.005,
  "SnipeAtPokestops": false,
  "SnipeIgnoreUnknownIv": false,
  "UseTransferIvForSnipe": false,
  "SnipePokemonNotInPokedex": true,
  "RenamePokemon": true,
  "RenameOnlyAboveIv": true,
  "RenameTemplate": "{1}_{0}",
  "MaxPokeballsPerPokemon": 6,
  "MaxTravelDistanceInMeters": 2205,
  "TotalAmountOfPokeballsToKeep": 194,
  "TotalAmountOfPotionsToKeep": 5,
  "TotalAmountOfRevivesToKeep": 1,
  "TotalAmountOfBerriesToKeep": 50,
  "UseGreatBallAboveCp": 1000,
  "UseUltraBallAboveCp": 1250,
  "UseMasterBallAboveCp": 1500,
  "UseGreatBallAboveIv": 85.0,
  "UseUltraBallAboveIv": 95.0,
  "UseGreatBallBelowCatchProbability": 0.2,
  "UseUltraBallBelowCatchProbability": 0.1,
  "UseMasterBallBelowCatchProbability": 0.05,
  "EnableHumanizedThrows": true,
  "EnableMissedThrows": false,
  "ThrowMissPercentage": 25,
  "NiceThrowChance": 40,
  "GreatThrowChance": 30,
  "ExcellentThrowChance": 10,
  "CurveThrowChance": 90,
  "ForceGreatThrowOverIv": 90.0,
  "ForceExcellentThrowOverIv": 95.0,
  "ForceGreatThrowOverCp": 1000,
  "ForceExcellentThrowOverCp": 1500,
  "TransferWeakPokemon": true,
  "TransferDuplicatePokemon": true,
  "TransferDuplicatePokemonOnCapture": true,
  "FavoriteMinIvPercentage": 99.0,
  "AutoFavoritePokemon": true,
  "UsePokemonToNotCatchFilter": false,
  "UsePokemonSniperFilterOnly": false,
  "ItemRecycleFilter": [
    {
      "Key": "itemUnknown",
      "Value": 0
    },
    {
      "Key": "itemLuckyEgg",
      "Value": 200
    },
    {
      "Key": "itemIncenseOrdinary",
      "Value": 100
    },
    {
      "Key": "itemIncenseSpicy",
      "Value": 100
    },
    {
      "Key": "itemIncenseCool",
      "Value": 100
    },
    {
      "Key": "itemIncenseFloral",
      "Value": 100
    },
    {
      "Key": "itemTroyDisk",
      "Value": 100
    },
    {
      "Key": "itemXAttack",
      "Value": 100
    },
    {
      "Key": "itemXDefense",
      "Value": 100
    },
    {
      "Key": "itemXMiracle",
      "Value": 100
    },
    {
      "Key": "itemSpecialCamera",
      "Value": 100
    },
    {
      "Key": "itemIncubatorBasicUnlimited",
      "Value": 100
    },
    {
      "Key": "itemIncubatorBasic",
      "Value": 100
    },
    {
      "Key": "itemPokemonStorageUpgrade",
      "Value": 100
    },
    {
      "Key": "itemItemStorageUpgrade",
      "Value": 100
    },
    {
      "Key": "itemUnknown",
      "Value": 0
    },
    {
      "Key": "itemLuckyEgg",
      "Value": 200
    },
    {
      "Key": "itemIncenseOrdinary",
      "Value": 100
    },
    {
      "Key": "itemIncenseSpicy",
      "Value": 100
    },
    {
      "Key": "itemIncenseCool",
      "Value": 100
    },
    {
      "Key": "itemIncenseFloral",
      "Value": 100
    },
    {
      "Key": "itemTroyDisk",
      "Value": 100
    },
    {
      "Key": "itemXAttack",
      "Value": 100
    },
    {
      "Key": "itemXDefense",
      "Value": 100
    },
    {
      "Key": "itemXMiracle",
      "Value": 100
    },
    {
      "Key": "itemSpecialCamera",
      "Value": 100
    },
    {
      "Key": "itemIncubatorBasicUnlimited",
      "Value": 100
    },
    {
      "Key": "itemIncubatorBasic",
      "Value": 100
    },
    {
      "Key": "itemPokemonStorageUpgrade",
      "Value": 100
    },
    {
      "Key": "itemItemStorageUpgrade",
      "Value": 100
    },
    {
      "Key": "itemUnknown",
      "Value": 0
    },
    {
      "Key": "itemLuckyEgg",
      "Value": 200
    },
    {
      "Key": "itemIncenseOrdinary",
      "Value": 100
    },
    {
      "Key": "itemIncenseSpicy",
      "Value": 100
    },
    {
      "Key": "itemIncenseCool",
      "Value": 100
    },
    {
      "Key": "itemIncenseFloral",
      "Value": 100
    },
    {
      "Key": "itemTroyDisk",
      "Value": 100
    },
    {
      "Key": "itemXAttack",
      "Value": 100
    },
    {
      "Key": "itemXDefense",
      "Value": 100
    },
    {
      "Key": "itemXMiracle",
      "Value": 100
    },
    {
      "Key": "itemSpecialCamera",
      "Value": 100
    },
    {
      "Key": "itemIncubatorBasicUnlimited",
      "Value": 100
    },
    {
      "Key": "itemIncubatorBasic",
      "Value": 100
    },
    {
      "Key": "itemPokemonStorageUpgrade",
      "Value": 100
    },
    {
      "Key": "itemItemStorageUpgrade",
      "Value": 100
    },
    {
      "Key": "itemUnknown",
      "Value": 0
    },
    {
      "Key": "itemLuckyEgg",
      "Value": 200
    },
    {
      "Key": "itemIncenseOrdinary",
      "Value": 100
    },
    {
      "Key": "itemIncenseSpicy",
      "Value": 100
    },
    {
      "Key": "itemIncenseCool",
      "Value": 100
    },
    {
      "Key": "itemIncenseFloral",
      "Value": 100
    },
    {
      "Key": "itemTroyDisk",
      "Value": 100
    },
    {
      "Key": "itemXAttack",
      "Value": 100
    },
    {
      "Key": "itemXDefense",
      "Value": 100
    },
    {
      "Key": "itemXMiracle",
      "Value": 100
    },
    {
      "Key": "itemSpecialCamera",
      "Value": 100
    },
    {
      "Key": "itemIncubatorBasicUnlimited",
      "Value": 100
    },
    {
      "Key": "itemIncubatorBasic",
      "Value": 100
    },
    {
      "Key": "itemPokemonStorageUpgrade",
      "Value": 100
    },
    {
      "Key": "itemItemStorageUpgrade",
      "Value": 100
    }
  ],
  "PokemonsNotToTransfer": [
    "ivysaur",
    "venusaur",
    "charmander",
    "charizard",
    "wartortle",
    "blastoise",
    "butterfree",
    "beedrill",
    "pikachu",
    "raichu",
    "nidoqueen",
    "nidoking",
    "clefable",
    "ninetales",
    "vileplume",
    "poliwrath",
    "alakazam",
    "machamp",
    "victreebel",
    "golem",
    "slowbro",
    "magneton",
    "farfetchd",
    "muk",
    "cloyster",
    "gengar",
    "electrode",
    "exeggutor",
    "weezing",
    "chansey",
    "kangaskhan",
    "mrMime",
    "gyarados",
    "lapras",
    "ditto",
    "porygon",
    "omastar",
    "kabutops",
    "snorlax",
    "articuno",
    "zapdos",
    "moltres",
    "dragonair",
    "dragonite",
    "mewtwo",
    "mew"
  ],
  "PokemonsToEvolve": [
    "bulbasaur",
    "ivysaur",
    "charmander",
    "charmeleon",
    "squirtle",
    "wartortle",
    "caterpie",
    "metapod",
    "weedle",
    "kakuna",
    "pidgey",
    "pidgeotto",
    "rattata",
    "spearow",
    "ekans",
    "pikachu",
    "sandshrew",
    "nidoranFemale",
    "nidorina",
    "nidoranMale",
    "nidorino",
    "clefairy",
    "jigglypuff",
    "zubat",
    "oddish",
    "gloom",
    "paras",
    "venonat",
    "diglett",
    "meowth",
    "psyduck",
    "mankey",
    "growlithe",
    "poliwag",
    "poliwhirl",
    "kadabra",
    "machop",
    "machoke",
    "bellsprout",
    "weepinbell",
    "victreebel",
    "tentacool",
    "geodude",
    "graveler",
    "ponyta",
    "slowpoke",
    "magnemite",
    "doduo",
    "seel",
    "grimer",
    "shellder",
    "gastly",
    "haunter",
    "drowzee",
    "krabby",
    "voltorb",
    "exeggcute",
    "cubone",
    "koffing",
    "rhyhorn",
    "horsea",
    "goldeen",
    "staryu",
    "magikarp",
    "eevee",
    "omanyte",
    "kabuto",
    "dratini",
    "dragonair"
  ],
  "PokemonsToLevelUp": [
    "venusaur",
    "charizard",
    "blastoise",
    "clefable",
    "muk",
    "chansey",
    "gyarados",
    "lapras",
    "ditto",
    "snorlax",
    "articuno",
    "zapdos",
    "moltres",
    "dragonite",
    "mewtwo",
    "mew"
  ],
  "PokemonsToIgnore": [
    "caterpie",
    "weedle",
    "pidgey",
    "rattata",
    "spearow",
    "zubat",
    "doduo"
  ],
  "PokemonsTransferFilter": {
    "Golduck": {
      "KeepMinCp": 1800,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Farfetchd": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 80.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Krabby": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Kangaskhan": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 60.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Horsea": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Staryu": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "MrMime": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 40.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Scyther": {
      "KeepMinCp": 1800,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 80.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Jynx": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Electabuzz": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 80.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Magmar": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 80.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Pinsir": {
      "KeepMinCp": 1800,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Tauros": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Magikarp": {
      "KeepMinCp": 200,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Gyarados": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Lapras": {
      "KeepMinCp": 1800,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 80.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Eevee": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Vaporeon": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Jolteon": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Flareon": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Porygon": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 60.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Snorlax": {
      "KeepMinCp": 2600,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Dragonite": {
      "KeepMinCp": 2600,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [
        []
      ],
      "DeprecatedMoves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    }
  },
  "PokemonToSnipe": {
    "Locations": [
      {
        "Latitude": 38.556807486461118,
        "Longitude": -121.2383794784546
      },
      {
        "Latitude": -33.859019,
        "Longitude": 151.213098
      },
      {
        "Latitude": 47.5014969,
        "Longitude": -122.0959568
      },
      {
        "Latitude": 51.5025343,
        "Longitude": -0.2055027
      }
    ],
    "Pokemon": [
      "venusaur",
      "charizard",
      "blastoise",
      "beedrill",
      "raichu",
      "sandslash",
      "nidoking",
      "nidoqueen",
      "clefable",
      "ninetales",
      "golbat",
      "vileplume",
      "golduck",
      "primeape",
      "arcanine",
      "poliwrath",
      "alakazam",
      "machamp",
      "golem",
      "rapidash",
      "slowbro",
      "farfetchd",
      "muk",
      "cloyster",
      "gengar",
      "exeggutor",
      "marowak",
      "hitmonchan",
      "lickitung",
      "rhydon",
      "chansey",
      "kangaskhan",
      "starmie",
      "mrMime",
      "scyther",
      "magmar",
      "electabuzz",
      "jynx",
      "gyarados",
      "lapras",
      "ditto",
      "vaporeon",
      "jolteon",
      "flareon",
      "porygon",
      "kabutops",
      "aerodactyl",
      "snorlax",
      "articuno",
      "zapdos",
      "moltres",
      "dragonite",
      "mewtwo",
      "mew"
    ]
  },
  "PokemonToUseMasterball": [
    "articuno",
    "zapdos",
    "moltres",
    "mewtwo",
    "mew"
  ]
}
```

### Steps to Reproduce
Tried quitting NecroBot, changing config to display 20 then restart and still no results.  Changed back to 10 and still blank.

### Other Information
Version: v0.8.8
Windows Version:  Windows 10 Pro
What is affected (catch, transfer, evolve, recycle, snipe, powerup/levelup, other):  Other - not displaying highest CP/IV (Perfect) Pokemon at startup.

**Comment by NoxxDev** [comment]
Has been fixed in the current version 0.8.8
Delete all files of the bot [download] (https://github.com/NoxxDev/NecroBot/releases/tag/v0.8.8) and reconfigure

![](http://puu.sh/qB7IE/9b859ef452.png)

**Comment by Yombre** [comment]
Thank you!
